selfdrive/locationd/locationd.cc

```diff
@@ -711,9 +720,9 @@ int Localizer::locationd_thread() {
     // 100Hz publish for notcars, 20Hz for cars
     const char* trigger_msg = sm["carParams"].getCarParams().getNotCar() ? "accelerometer" : "cameraOdometry";
     if (sm.updated(trigger_msg)) {
-      bool inputsOK = sm.allAliveAndValid() && this->are_inputs_ok();
+      bool inputsOK = sm.allAliveAndValid();
       bool gpsOK = this->is_gps_ok();
-      bool sensorsOK = sm.allAliveAndValid({"accelerometer", "gyroscope"});
+      bool sensorsOK = sm.allAliveAndValid({"accelerometer", "gyroscope"}) && this->are_inputs_ok();
```

Comment:
doesn't this include a bunch of non-sensor stuff?

Reply:
It has `{"cameraOdometry", "liveCalibration", "accelerometer", "gyroscope"}`. 
All are necessary to be checked, IMO. Split this to sensor only for now?